templates/bot/ts/default/package.json

```diff
@@ -6,7 +6,7 @@
   "license": "MIT",
   "main": "./lib/index.js",
   "scripts": {
-    "build": "tsc --build && shx cp -r ./public ./lib/",
+    "build": "tsc --build",
```

Comment:
Nit: remove shx devDep too, since it's useless.

Reply:
ok